app/celer-dump-data.cc

```diff
@@ -78,25 +81,74 @@ void print_particles(ParticleParams const& particles)
 /*!
  * Print element properties.
  */
-void print_elements(std::vector<ImportElement>& elements)
+void print_elements(std::vector<ImportElement>& elements,
+                    std::vector<ImportIsotope>& isotopes)
 {
     CELER_LOG(info) << "Loaded " << elements.size() << " elements";
     cout << R"gfm(
 # Elements
 
-| Element ID | Name | Atomic number | Mass (AMU) |
-| ---------- | ---- | ------------- | ---------- |
+| Element ID | Name | Atomic number | Mass (AMU) | Isotopes                                 | Relative abundance                                           |
+| ---------- | ---- | ------------- | ---------- | ---------------------------------------- | ------------------------------------------------------------ |
```

Comment:
We don't bother printing the number fractions for elements in materials; should we do the same there? Or have a separate (more compact) table that has one row per iso/frac and "merges" the element name cell? Maybe for now just kill the relative abundances.

Reply:
Fair. I'll remove the relative abundances. This will also resolve the previous question, about the `to_string` call (which was not really necessary, it was a remnant from a previous iteration that did not use stringstream).